group_vars/bibdata/staging.yml

```diff
@@ -141,14 +141,16 @@ postgresql_is_local: false
 
```

Comment:
Should datadog even be configured on this machine? This doesn't need to block this PR, just confused.

Reply:
@maxkadel Great point. I am also seeing several RDSS projects that don't need DataDog at all and I'll be removing it via a separate PR.